### DISCUSSION TITLE
How to set a tag name when using resticw?

### DISCUSSION THREAD

**Discussion by agingorange** [erikw/restic-automatic-backup-scheduler#93] [Q&A] [answered]
Is it possible to set a tag name when using `resticw`?

**Comment by erikw** [comment, marked as answer]
Hey,

`resticw` is a warper around `restic`, so any command should work. I'm not sure exactly what you mean with tag name? Do you mean to set the tags of a specific backup?

**Comment by agingorange** [comment]
Exactly that! I tried `--tag before_foo` but it didn't use that tag.

**Reply by erikw** [reply]
Could you show the full command that you used?

**Comment by agingorange** [comment]
`sudo resticw -p homes backup --tag new_setup /home`

I'm not sure why I have to add the path, as that is defined in `homes.env.sh`, so I might not be using it correctly.

**Reply by erikw** [reply]
The intention is that the script [restic_backup.sh](https://github.com/erikw/restic-automatic-backup-scheduler/blob/main/bin/restic_backup.sh) will do all the backing up. It will set the tags etc. as configured in your profile. The script will use the backup paths you've configured.

See this section in the README. Source the config profile and then run the above script

https://github.com/erikw/restic-automatic-backup-scheduler#5-make-first-backup

**Reply by agingorange** [reply]
I'll do it that way. Funny thing is, I just tried the `--tag` again and it worked. Not sure what's happening here, but I must have done it incorrectly previously.
